### PR TITLE
Fix output on donation confirmation page

### PR DESCRIPTION
**Pull request by KaiNissen**
- fix anonymous donations' address output
- fix translation of status codes

### CODE REVIEW

**Comment by KaiNissen** [comment]
[phab:T140120](https://phabricator.wikimedia.org/T140120)

**Comment by JeroenDeDauw** [comment]
Looks good to me. But the merge button is being all passive aggressive on me.

**Comment by KaiNissen** [comment]
refactored

File: tests/EdgeToEdge/Routes/ShowDonationConfirmationRouteTest.php [hunk 1]
```diff
@@ -125,23 +101,48 @@ public function testGivenAlternativeConfirmationPageConfig_alternativeContentIsD
...
+		$this->createEnvironment( [], function ( Client $client, FunFunFactory $factory ) {
+			$donation = $this->newBookedAnonymousPayPalDonation( $factory );
+
+			$responseContent = $this->retrieveDonationConfirmation( $client, $donation->getId() );
+
+			$this->assertContains( 'Ihre Spende per PayPal wurde gebucht', $responseContent );
+		} );
+	}
+
+	private function retrieveDonationConfirmation( Client $client, int $donationId ) {
```
Comment (JeroenDeDauw) [inline tests/EdgeToEdge/Routes/ShowDonationConfirmationRouteTest.php]: I'd have put this one after the first method using it, so less vertical navigation is needed when reading that one.

**Pull request merged**